src/app/debug/debug-tree/debug-tree.component.ts

```diff
@@ -48,79 +47,70 @@ export class DebugTreeComponent implements AfterViewInit, OnDestroy {
     this.showMultipleAtATimeSubscription.unsubscribe();
   }
 
-  subscribeToSettingsServiceObservables(): void {
-    this.showMultipleAtATimeSubscription = this.settingsService.showMultipleAtATimeObservable.subscribe(
-      (value: boolean) => {
-        this.showMultipleAtATime = value;
-        if (!this.showMultipleAtATime) {
-          this.removeAllReportsButOne();
-        }
-      },
-    );
+  @Input() set currentView(value: any) {
```

Comment:
Shall we create an issue to create a type for this?

Reply:
Yeah that seems right

Reply:
https://github.com/wearefrank/ladybug-frontend/issues/383#issue-2241600035